app/views/your-psp/_worldpay-flex.njk

```diff
@@ -13,7 +13,7 @@
     }) }}
 
     <p class="govuk-body">
-      If you have set up 3DS Flex on your Worldpay account, you need to enter the following details to turn 3DS Flex on.
+      To set up 3DS Flex on your Worldpay test account, you need to enter the following test details to turn 3DS Flex on.
       You’ll find these details in your <a class="govuk-link" href="https://secure.worldpay.com/sso/public/auth/login.html">Worldpay account</a>.
```

Comment:
The design has updated so that line 17 is not shown for LIVE accounts.

I thought it would have been worth keeping for test accounts as they could toggle 3DS.

How about the following:

## Live will continue to look like the new design:
<img width="739" alt="image" src="https://github.com/alphagov/pay-selfservice/assets/59831992/47685b20-532f-4a8e-bf13-2b9bf50cbb51">

## Test Worldpay accounts will see the following:

<img width="739" alt="image" src="https://github.com/alphagov/pay-selfservice/assets/59831992/6c20b248-0e6b-459f-a77d-33078bbe701a">

Reply:
A cool, makes sense. I don't think we should have it for test accounts either.

Reply:
Yeh I think we could just remove the hint text, don't think it's helping anyone

Reply:
Hmm although maybe it makes it clear you need to enter the credentials, then toggle the button

Reply:
Maybe just remove line 17?

Reply:
Thanks - I will remove line 17.

Reply:
Just updated it - thansk